update README.md (#277)
ToruNiina/toml11: README.md

}
```

> Fix syntax errors in examples
> Fix syntax errors in examplesFor more details, please refer to the [documentation](https://toruniina.github.io/toml11/).

Reviewer: ```suggestion
For more details, please refer to the [documentation](https://toruniina.github.io/toml11/).
```
